utils/das_data_sorter.py

```diff
@@ -1,13 +1,14 @@
 import pandas as pd 
 from argparse import ArgumentParser
-from numpy import ceil
+from numpy import ceil, median
 
 # accepts terminal arguments
 parser = ArgumentParser()
 parser.add_argument("--input", help="Reads the inputted CSV file to filter", action="store", required=True)
 parser.add_argument("--output", help="Writes the filtered data onto a new CSV file under this name", action="store", required=True)
 parser.add_argument("--unit", help="Specify time units [seconds, s, or minutes, m]. Default is in seconds.", default="seconds", 
                     choices=["seconds", "s", "minutes", "m"], action="store")
+parser.add_argument("--smooth", help="Smooths data points using 3-point mean or median smoothing", choices=["mean", "median"], action="store")
```

Comment:
You say 3-point mean/median here - do you plan be able to do other numbers of points?

Reply:
Oh yes, I was planning to have an input for other numbers of points. That's another typo 😅